### PR TITLE
[WebInterface] Add trading state change

### PR DESCRIPTION
**Pull request by Herklos**
image
image

image
image

image

image

### CODE REVIEW

File: Services/Interfaces/web_interface/templates/components/modals/trading_state_modal.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+              <span aria-hidden="true">&times;</span>
+            </button>
+          </div>
+          <div class="modal-body">
+              <p>
+                {% if is_real_trading %}
+                    By switching to simulated trading, OctoBot will only use its simulation mode on real market conditions.
+                    It will no longer create trade with your exchange account, it will use a simulated portfolio managed by OctoBot.
+
+                    <div class="my-4 text-center">
```
[the remark this answers is not in the document]
Comment (Herklos) [reply Services/Interfaces/web_interface/templates/components/modals/trading_state_modal.html:18]: You mean the same as real trading switch : "Stay" ?

[the remark this answers is not in the document]
Comment (GuillaumeDSM) [reply Services/Interfaces/web_interface/templates/components/modals/trading_state_modal.html:18]: yes

**Comment by Herklos** [comment]
After update :
image
image

image
image
image

**Review by GuillaumeDSM — APPROVED** [review]
🎉 great !

**Pull request merged**